plugin/plugin.go

```diff
@@ -164,6 +164,13 @@ type PluginContext interface {
 	// Region returns the targeted region
 	CurrentRegion() models.Region
 
+	// CRIType returns the type of compute resource the user logged in as, if applicable. Valid values are `IKS`, `VPC`, or `OTHER`
+	CRIType() string
+
+	// VPCCRITokenURL() returns the value specified by the environment variable 'IBMCLOUD_CR_VPC_URL', if set.
+	// Otherwise, the default VPC auth url "http://169.254.169.254" is returned
```

Comment:
Rather than including this URL in the comment, this should just point to the constant. Just in case this value ever changes for some reason, it is only updated in one place, then.